test/files/mysql/schema.sql

```diff
@@ -181,7 +181,7 @@ create table type_monsters (
     generated_nnull text GENERATED ALWAYS AS (UPPER(text_nnull)) STORED NOT NULL,
     generated_null text GENERATED ALWAYS AS (UPPER(text_null)) STORED,
 
-    INDEX(int_one, int_two)
+    UNIQUE(int_one, int_two)
```

Comment:
Why is the change needed 🤔 ?

Reply:
Running the tests locally on MySQL 8.4.0 was producing the following error:

```
=== RUN   TestDriver
dropping database... DONE
creating database... DONE
selecting database... DONE
migrating...migrating mysql/schema.sql...
    mysql_test.go:74: migrating mysql/schema.sql: Error 6125 (HY000): Failed to add the foreign key constraint. Missing unique key for constraint 'multi_keys_ibfk_1' in the referenced table 'type_monsters'
--- FAIL: TestDriver (0.16s)

FAIL

Process finished with the exit code 1
```